src/app/client/src/app/modules/player-helper/components/player/player.component.ts

```diff
@@ -28,19 +32,47 @@ export class PlayerComponent implements AfterViewInit, OnChanges {
   CONSTANT = {
     ACCESSEVENT: 'renderer:question:submitscore'
   };
+
+  @Input() overlayImagePath: string;
+  @Input() isSingleContent: boolean;
+  @Input() telemetryObject: {};
```

Comment:
telemetryObject will itself having the pageID, why do you want to take another pageId as input.

Reply:
It won't have